### ISSUE TITLE
NPE on news view model when checking for updates

### ISSUE THREAD

**Issue by RyanTheAllmighty** [ATLauncher/ATLauncher#856]
### What happened?

After exiting Minecraft, the dialog pops up checking for updates, then error in the console.

I couldn't replicate it, but the error is clear and something we can just null check before calling

### Operating System

Windows

### Relevant log output

```shell
[02/12/2023 09:16:55 am] [Render thread/INFO] Stopping!
[02/12/2023 09:16:56 am] Checking for updated files!
[02/12/2023 09:16:56 am] Checking for updates to CurseForge instances
[02/12/2023 09:16:56 am] Checking for updated files!
[02/12/2023 09:16:57 am] Finished downloading updated files!
[02/12/2023 09:16:57 am] Checking for updates to CurseForge instances
[02/12/2023 09:16:58 am] java.lang.NullPointerException: Cannot invoke "com.atlauncher.viewmodel.base.INewsViewModel.reload()" because "this.viewModel" is null
[02/12/2023 09:16:58 am] 	at com.atlauncher.gui.tabs.news.NewsTab.reload(NewsTab.java:155)
[02/12/2023 09:16:58 am] 	at com.atlauncher.Launcher.reloadNewsPanel(Launcher.java:476)
[02/12/2023 09:16:58 am] 	at com.atlauncher.Launcher.lambda$reloadLauncherData$11(Launcher.java:380)
[02/12/2023 09:16:58 am] 	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
[02/12/2023 09:16:58 am] 	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
[02/12/2023 09:16:58 am] 	at java.base/java.lang.Thread.run(Unknown Source)
[02/12/2023 09:16:58 am] java.lang.NullPointerException: Cannot invoke "com.atlauncher.viewmodel.base.INewsViewModel.reload()" because "this.viewModel" is null
	at com.atlauncher.gui.tabs.news.NewsTab.reload(NewsTab.java:155)
	at com.atlauncher.Launcher.reloadNewsPanel(Launcher.java:476)
	at com.atlauncher.Launcher.lambda$reloadLauncherData$11(Launcher.java:380)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
	at java.base/java.lang.Thread.run(Unknown Source)
```

### Is this actually a launcher bug?

- [X] I am sure this is a bug with the launcher and not an issue with running Minecraft

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

**Comment by Doomsdayrs** [comment]
LifeCycle issue, working on it now.

**Comment by RyanTheAllmighty** [comment]
Fixed with version 3.4.35.4
